src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/UpdateAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -31,6 +32,23 @@ public class SetAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
             ValueFromPipeline = true,
             HelpMessage = "The VirtualNetworkGatewayConnection")]
         public PSVirtualNetworkGatewayConnection VirtualNetworkGatewayConnection { get; set; }
+        
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Whether to establish a BGP session over a S2S VPN tunnel")]
+        public string EnableBgp { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Use policy-based traffic selectors for a S2S connection")]
+        public string UsePolicyBasedTrafficSelectors { get; set; }
+
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "A list of IPSec policies.")]
+        public List<PSIpsecPolicy> IpsecPolicies { get; set; }
```

Comment:
@henry416 this parameter name should be `IpsecPolicy`. From the [Strongly Encouraged Developer Guidelines](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx):

_Avoid using plural names for parameters whose value is a single element. This includes parameters that take arrays or lists because the user might supply an array or list with only one element._